style.css

```diff
@@ -96,22 +101,254 @@ header {
   margin-top: 20px;
 }
 
-.fa-linkedin {
+.fa-linkedin,
+.fa-github,
+.fa-medium,
+.fa-twitter,
+.fa-angellist {
   color: #505f79;
 }
 
-.fa-github {
-  color: #505f79;
+.card {
+  display: flex;
+  flex-direction: column;
+  border-radius: 16px;
+  border: solid #e5e5e5 1px;
+  background-color: #fff;
+  justify-content: center;
+  padding: 16px;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-bottom: 40px;
 }
 
-.fa-medium {
-  color: #505f79;
+.snapshot {
+  width: 100%;
 }
 
-.fa-twitter {
-  color: #505f79;
+h2 {
+  width: 295px;
+  height: 36px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
 }
 
-.fa-angellist {
+.detail {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  padding: 0;
+}
+
+.main {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  margin-right: 10px;
+  padding-right: 12px;
+}
+
+.main2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  margin-right: 10px;
+  padding-right: 12px;
+  float: left;
+  list-style-type: disc;
+  color: #7a869a;
+}
+
+.paragraph {
+  margin-top: 15px;
+  padding-right: 16px;
+}
+
+.shor-descr {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #344563;
+}
+
+.language {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  flex-direction: row;
+  text-align: center;
+  margin-bottom: 36px;
+  margin-top: 12px;
+  padding-left: 0;
+  gap: 12px;
+}
+
+.language li {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+  padding-top: 6px;
+  padding-bottom: 6px;
+  padding-left: 10px;
+  padding-right: 10px;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+.Project {
+  margin-top: 114px;
+  margin-left: 10px;
+  margin-right: 10px;
+  display: grid;
+  grid-template-columns: 1fr 2fr 1fr;
+  grid-template-rows: auto;
+  gap: 10px;
+}
```

Comment:
- [ ] The reason that this style wasn't applying to your "Projects" section is because of a typo, you set the class of that section to ".Projects" but in your style, you are referring to is as ".Project" (you missed the "s"), please before merging this PR make sure to fix this typo.

Reply:
Thank you so much.